tests/Microsoft.Bot.Builder.Dialogs.Declarative.Tests/JsonLoadTests.cs

```diff
@@ -11,46 +11,41 @@
 using Microsoft.Bot.Builder.Dialogs.Adaptive;
 using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
 using Microsoft.Bot.Schema;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Xunit;
 
 namespace Microsoft.Bot.Builder.Dialogs.Loader.Tests
 {
-    [TestClass]
-    public class JsonLoadTests
+    public class JsonLoadTests : IDisposable
     {
-        private static ResourceExplorer resourceExplorer;
+        private static ResourceExplorer _resourceExplorer;
 
-        public TestContext TestContext { get; set; }
-
-        [ClassInitialize]
```

Comment:
I think we should use IClassFixture replacing ClassInitialize so it is executed once before running all the tests in this class.

Reply:
Feedback applied. Thanks!